.travis.yml

```diff
@@ -13,18 +13,18 @@ cache:
 
 matrix:
   include:
-    - env: GCC_VERSION=6 BUILD_TYPE=Debug ASAN=Off
-      os: linux
-      addons: &gcc6
-        apt:
-          packages:
-            - g++-6
-          sources:
-            - ubuntu-toolchain-r-test
+    # - env: GCC_VERSION=6 BUILD_TYPE=Debug ASAN=Off
```

Comment:
Please remove this commented-out GCC6 stuff.

Reply:
Will do. And don't merge yet. I'm bringing in the `subrange` changes from #126.